java/generate-offline-map-overrides/src/main/java/com/esri/arcgisruntime/generateofflinemapoverrides/MainActivity.java

```diff
@@ -61,22 +51,26 @@
 import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
 import com.esri.arcgisruntime.tasks.geodatabase.GenerateGeodatabaseParameters;
 import com.esri.arcgisruntime.tasks.geodatabase.GenerateLayerOption;
-import com.esri.arcgisruntime.tasks.offlinemap.GenerateOfflineMapJob;
-import com.esri.arcgisruntime.tasks.offlinemap.GenerateOfflineMapParameterOverrides;
-import com.esri.arcgisruntime.tasks.offlinemap.GenerateOfflineMapParameters;
-import com.esri.arcgisruntime.tasks.offlinemap.GenerateOfflineMapResult;
-import com.esri.arcgisruntime.tasks.offlinemap.OfflineMapParametersKey;
-import com.esri.arcgisruntime.tasks.offlinemap.OfflineMapTask;
+import com.esri.arcgisruntime.tasks.offlinemap.*;
 import com.esri.arcgisruntime.tasks.tilecache.ExportTileCacheParameters;
 
+import java.io.File;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.ExecutionException;
+
 public class MainActivity extends AppCompatActivity {
 
   private static final String TAG = MainActivity.class.getSimpleName();
 
   private MapView mMapView;
+
```

Comment:
And added white space here (usually we clump them (in the same order as the imports):

java
android
runtime